ramls/checkout-lock-storage.raml

```diff
@@ -0,0 +1,95 @@
+#%RAML 1.0
+title: Checkout lock Storage
+version: v1.0
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: Checkout lock API
```

Comment:
It's not the most important issue, but I still think consistency is a god thing, especially considering that API docs will be created based on these RAML files and JSON schemas. If you check the descriptions in this RAML file and JSON schema files you will find that the same thing is referred to in many different ways:
- checkout lock
- check out lock
- Checkout Lock
- check-out lock
- checkoutLock
- lock
Personally, I would try to choose one way of calling it and stick to it throughout all descriptions.

Reply:
Checkout Lock - I followed this format and made the changes wherever it is applicable